packages/test/end-to-end-tests/package.json

```diff
@@ -31,7 +31,8 @@
     "test:realsvc:report": "cross-env FLUID_TEST_REPORT=1 npm run test:realsvc",
     "test:realsvc:routerlicious": "cross-env FLUID_TEST_DRIVER=routerlicious npm run test:realsvc:run",
     "test:realsvc:run": "mocha dist/test --config src/test/.mocharc.js",
-    "test:realsvc:tinylicious": "cross-env FLUID_TEST_DRIVER=tinylicious start-server-and-test start:tinylicious:test 3000 test:realsvc:run",
+    "test:realsvc:tinylicious": "cross-env FLUID_TEST_DRIVER=tinylicious start-server-and-test start:tinylicious:test 3000 test:realsvc:tinylicious:run",
```

Comment:
this is fine, but we could also add an env var to specificy parallel count, and process it in the config. this would reduce script entries

Reply:
yeah... I might change that in the future.